docs/integrations/sources/monday-migrations.md

```diff
@@ -0,0 +1,9 @@
+# Monday Migration Guide
+
+## Upgrading to 2.0.0
```

Comment:
please follow the migration guide template: https://docs.google.com/document/d/1V3ybA24Lj_fy7IQwsVVzSxs46qGOsUydwlAQEYSQfcE/edit?pli=1

Reply:
Done